config/routes.rb

```diff
@@ -662,9 +662,8 @@
       get '/timeline' => 'application_choices#timeline', as: :application_choice_timeline
       get '/emails' => 'application_choices#emails', as: :application_choice_emails
       get '/feedback' => 'application_choices#feedback', as: :application_choice_feedback
-      get '/conditions' => 'conditions#edit', as: :application_choice_edit_conditions
-      patch '/conditions/confirm' => 'conditions#confirm_update', as: :application_choice_confirm_update_conditions
-      patch '/conditions' => 'conditions#update', as: :application_choice_update_conditions
+      # TODO: Remove redirect after 1 Sept 2021
+      get '/conditions' , to: redirect('/provider/applications/%{application_choice_id}/offer/conditions')
```

Comment:
Should this redirect to `/id/condition-statuses/edit`?

Reply:
Done. I grepped the routes for `conditions` and missed that path.